config.go

```diff
@@ -112,10 +119,6 @@ func (m *Defender) Validate() error {
 	if m.responder == nil {
 		return fmt.Errorf("responder not configured")
 	}
-	if len(m.Ranges) == 0 {
```

Comment:
By using all predefined ranges, wouldn't this include private and essentially block all access for most if `ranges` is omitted?

Reply:
Yes. It's not a perfect solution. If you have any suggestions for an alternative let me know

Reply:
Yes. It's not a perfect solution. If you have any suggestions for an alternative let me know